collection/10.5281/zenodo.5764892/resource.yaml

```diff
@@ -1,9 +1,21 @@
 doi: 10.5281/zenodo.5764892
 id: 10.5281/zenodo.5764892
-owners: [77626]
```

Comment:
these formatting changes are an artifact of developing this repo and should not occur in the future / after a reset of gh-pages...